lib/gui/models/store.js

```diff
@@ -94,15 +95,15 @@ const storeReducer = (state, action) => {
 
         const drive = _.first(action.data);
 
-        if (_.every([
-
-          // TODO: Reuse from SelectionStateModel.isDriveValid()
-          state.getIn([ 'selection', 'image', 'size' ], 0) <= drive.size,
+        // Even if there's no image selectected, we need to call `isDriveValid`
+        // to check if the drive is locked, and `{}` works fine with it
+        const image = state.getIn([ 'selection', 'image' ], {
```

Comment:
> This would prevent us from accidentally relying on an empty object representing the lack of an image in the store.

We're not relying on an empty object being the store. If the store returns something, we want to call `toJS()` on that something. If it returns `undefined`, we want to not call `toJS()` and just use `{}`

Your version is more readable, except we still need to call `toJS()` on image if it's not `undefined`.